pages/common/npx.md

```diff
@@ -0,0 +1,15 @@
+# npx
+
+> Execute binaries from `npm` packages(most of the cli tools) .
+
+- Execute the npm packages binaries(Package will be installed if it does not exist in node_modules or $PATH):
+
+`npx {{module_name}}`
+
+- Want to use multiple binaries, define the package name to be installed, then execute the binaries:
```

Comment:
This sentence is not grammatically correct. How about `In case a package has multiple binaries, specify the package name along with the binary` ?

Reply:
Thx